README.md

```diff
@@ -68,6 +68,9 @@ apt install gcc linux-headers-generic make patch wget
 ```
 **arch package install**
 ```
+# Install from the AUR
+yay -S linux-headers snd-hda-macbookpro-dkms-git
```

Comment:
I'm new to AUR. Can `linux-headers` be a dependency for `snd-hda-macbookpro-dkms-git` instead of having to explicitly list both?

Reply:
tl;dr: No, probably not.

Thanks for testing the PKGBUILD, and for the input! I didn't include `linux-headers` as a dependency due to the specific guideline on this in the [ArchWiki page on DKMS packages](https://wiki.archlinux.org/title/DKMS_package_guidelines#Dependencies), though I'll admit it's not a topic I'm very familiar with. The idea seems to be that there are multiple variations of the `linux-headers` package, and so including one of them as a dependency has the potential to cause conflicts I guess. Including it as an explicit package in the install command is a little inelegant, but in theory it should JustWork™ as is for the majority of users, while users that know more about the different header packages will know how to modify the command appropriately.

One potential problem with command right now is that `linux-headers` is being installed with "explicitly requested" as the install reason, rather than as a "dependency". If that is what you are concerned about, I think it could be resolved by modifying the install command to be:
`yay -S --asdeps linux-headers --asexplicit snd-hda-macbookpro-dkms-git # I haven't tested this yet`,
but again, similar to the different `linux-headers`, I think potentially this is a point that most users are not concerned about (based on a sample size of only me), and the ones that are will already know how to change `linux-headers` to be a dependency. It could be that I'm wrong about people's preferences on `--asdeps`, so if you think that it would make more sense like that, I'm fine changing the install command to something like the above.